glyphs-reader/src/font.rs

```diff
@@ -520,7 +527,15 @@ impl RawCustomParameters {
         {
             macro_rules! add_and_report_issues {
```

Comment:
Not a huge fan of the macro, lets at least add a comment explaining what precludes doing this without a macro